src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+  &.stars--1 .stars__star:nth-of-type(-n + 1),
+  &.stars--2 .stars__star:nth-of-type(-n + 2),
+  &.stars--3 .stars__star:nth-of-type(-n + 3),
+  &.stars--4 .stars__star:nth-of-type(-n + 4),
+  &.stars--5 .stars__star:nth-of-type(-n + 5) {
```

Comment:
The class names used for the star modifiers (e.g., 'stars--1') should follow the BEM naming convention more closely. According to the BEM methodology, modifiers should be separated by double dashes and should describe the state of the block or element. In this case, it seems like you are trying to indicate the number of active stars, but the naming could be more descriptive. Consider renaming the modifiers to reflect their purpose more clearly, such as 'stars--rating-1', 'stars--rating-2', etc., to indicate the rating value represented by the active stars.